source/fundamentals/transactions.txt

```diff
@@ -15,38 +13,35 @@ Transactions
 Overview
 --------
 
-Read this guide to learn how to perform **transactions** in MongoDB using the
-Node.js driver. A transaction is a unit of work, composed of a series of
-operations that you want either to succeed together, or fail together when one
-or more of the operations fail. This behavior is called **atomicity**.
-Atomicity is a property in which transactions composed of one or more
-operations occur all at once, such that no other client can observe them as
-separate operations, and that it leaves no changes if one of the operations
-fails.
+In this guide, you can learn how to use **transactions** with the
+{+driver-short+}. Transactions allow you
+to run a series of operations that do not change any data until the
+entire transaction is committed. If any operation in the transaction fails, the 
+driver aborts the transaction and discards all data changes before they
+ever become visible. This behavior is called **atomicity**.
```

Comment:
S: If we want to get grossly nitpicky, atomicity is more of a property than a behavior...

```suggestion
ever become visible. This feature is called **atomicity**.
```